### DISCUSSION TITLE
Please make the 4k lane wider

### DISCUSSION THREAD

**Discussion by Alsen99** [ppy/osu#21889] [Ideas] [answered]
I'm playing on a 15 inch laptop and the 4k lane is small compared to skins like inteliser use (Light Blaster v1.7), please make it wider by default in pro or both version.

**Comment by mk56-spn** [comment]
There are plans to make the width customisable iirc. This is something the Devs are aware of

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/discussions/20622
